scripts/utility.py

```diff
@@ -1275,7 +1275,8 @@ def gather_cat_objects(
         elif abbr == "clan":
             out_set.update([x for x in Cat.all_cats_list if not (x.dead or x.outside or x.exiled)])
         elif abbr == "some_clan":
-            out_set.update([x for x in Cat.all_cats_list if not (x.dead or x.outside or x.exiled)])
+            out_set.update([x for x in Cat.all_cats_list if
+                            not (x.dead or x.outside or x.exiled)])  # are these supposed to be the same??
```

Comment:
```suggestion
            clan_cats = [x for x in Cat.all_cats_list if not (x.dead or x.outside or x.exiled)]
            out_set.update(sample(clan_cats, randint(1, round(len(clan_cats) / 4))))
```
This is a possible implementation that returns up to a quarter of the clan's cats. I don't know what we want to happen here, fwiw, but I wanted to note a possible solution